cpp/modules/math.gl/core/src/core.h

```diff
@@ -859,8 +878,21 @@ template <typename coord>
 auto Matrix4<coord>::makePerspective(coord fovy, coord aspect, coord near, coord far) -> Matrix4<coord> {
   auto f = 1.0 / tan(fovy / 2.0);
   auto nf = 1.0 / (near - far);
+
+  auto zero = static_cast<coord>(0);
```

Comment:
Do we really need a temp called `zero`? Why do we suddenly get such heavy type checking?